src/object_sync/hello.cpp

```diff
@@ -0,0 +1,50 @@
+#include "hello.hpp"
+
+  // If this was not defined within a namespace, it would be in the global scope.
+namespace object_sync {
+  
+  // Default constructor
```

Comment:
Not exactly. The "default constructor" is a C++ term: http://en.cppreference.com/w/cpp/language/default_constructor. In this case the underlying `HelloWorld` C++ is not actually being constructed yet. You are only calling static methods (that are accessible without constructing an instance of the object). However the convention for constructing JS objects is  to call `new SomeObject()` and that JS call will map to this C++ code. So, in a way, this is a default constructor, just not yet in the C++ sense. The default constructor in C++ would be called if you did, in C++ `auto obj = new HellowObject();`.